src/components/App.js

```diff
@@ -218,6 +219,13 @@ class App extends Component {
     })
   }
 
+  toggleDisplayPlaylist = () => {
+    const playList = document.querySelector('.track-list')
```

Comment:
You wouldn't normally use query selectors in React, but [`refs`](https://reactjs.org/docs/refs-and-the-dom.html) instead. But even that seems wrong here... I would have instead saved the state in the Playlist component, then calculate the classes inside of its `render` method. Something like 

```
<div className={`track-list ${this.state.isOpen ? 'open-track-list' : ''}`} >
```

Reply:
Yes, touching the DOM isn't favourable, I wasn't able to pipe the ref of playlist div to the App component, I'll try the state approach though. :smile:

Reply:
So how do I send a message from the App component to the Playlist component the the list is unfolded, I think I need to create state for this in the App component and pass it down to the Playlist component? @filoxo

Reply:
Yes, passing it down from the App component to the Playlist component as a prop is good too.

Reply:
Oh also forgot, usually that'll also require passing a `toggleOpen` method from App to Playlist as a prop in order to change that value from inside the playlist component. Just food for thought in case that's something you're looking to achieve.

Reply:
Yes! :tada: :smile: